Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -8256,12 +8272,11 @@ String consumeFontFormat(CSSParserTokenRange& range, bool rejectStringValues)
     auto& arg = args.consumeIncludingWhitespace();
     if (!args.atEnd())
         return nullString();
-    if (arg.type() != IdentToken && (rejectStringValues || arg.type() != StringToken))
-        return nullString();
-    auto format = arg.value().toString();
-    if (arg.type() == IdentToken && !FontCustomPlatformData::supportsFormat(format))
-        return nullString();
-    return format;
+    if (arg.type() == IdentToken && isFontFormatKeywordValid(arg.id()))
+        return arg.value().toString();
+    if (arg.type() == StringToken && !rejectStringValues)
```

Comment:
Do you know if strings should reject keywords that don't match `isFontFormatKeywordValid` ?

Reply:
I don't think so, since the syntax takes <String>, any string should be parsed, and then we should reject it at loading time.